docs/release-notes.md

```diff
@@ -11,9 +11,15 @@ meta:
 
 ## 6.5.0
 ### Bug Fixes
-* Faster loading time for the organization dashboard
```

Comment:
Can you move the `Bug Fixes` heading to the bottom please

Reply:
Good point. Made the edit.